### PR TITLE
Fix: role added to component__widget div (fixes #112)

### LINKED ISSUE #112: screen reader should read out aria-labelledby/label to be consistent with latest slider
### Subject of the issue/enhancement/features
To be consistent with the slider reading out the header/ariaQuestion when tabbing through the screen we should add role="region" to component__widget

ref: 
https://github.com/adaptlearning/adapt-contrib-slider/pull/210 

### Your environment
- FW v5.40.3
- Latest Firefox
- Windows, NVDA

### PR DESCRIPTION
**Pull request by joe-allen-89**
[//]: # (Please title your PR according to eslint commit conventions)
[//]: # (See https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint#eslint-convention for details)

[//]: # (Link the PR to the original issue)
#112 

[//]: # (Delete Fix, Update, New and/or Breaking sections as appropriate)
### Fix
* role added to component__widget div (#112)

ref: 
This is to be consistent to changes made to the slider - https://github.com/adaptlearning/adapt-contrib-slider/pull/210

### CODE REVIEW

File: templates/confidenceSlider.jsx [hunk 1]
```diff
@@ -69,6 +69,7 @@ export default function ConfidenceSlider (props) {
         ])}
         aria-labelledby={ariaQuestion ? null : (displayTitle || body || instruction) && `${_id}-header`}
         aria-label={ariaQuestion || null}
+        role='region'
```
Comment (oliverfoster) [inline templates/confidenceSlider.jsx:72]: Why [region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/group_role) and not [group](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/group_role)?

image

image

Comment (joe-allen-89) [reply templates/confidenceSlider.jsx:72]: Group has been used where there is a collection of items with the same functionality MCQ/textInput etc, I felt region was better suited as it's not a collection of items but still important and we do want users to navigate to the section easily.

Comment (oliverfoster) [reply templates/confidenceSlider.jsx:72]: Group is not explicitly for a collection of items with the same functionality, it's not like `role=list`. Group is more for a bounded section of interactive user interface. Group is more akin to the grouping of an "Address" or "Card details" section on an order form.

Region sections are more [landmark](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/landmark_role) orientated, they have their own shortcut key. Using region on some widgets and group on others will have unintended consequences with keyboard navigation when using a screen reader.

Comment (joe-allen-89) [reply templates/confidenceSlider.jsx:72]: Thanks for the info, I shall update to group to avoid any issues with keyboard nav.

**Review by oliverfoster — APPROVED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by guywillis — APPROVED** [review]
👍

**Pull request merged**